explorer/client/src/__tests__/e2e.test.ts

```diff
@@ -484,28 +484,28 @@ describe('End-to-end Tests', () => {
 
             expect(
                 await page.$eval(
-                    '#groupCollection > div:nth-child(1) > div:nth-child(1)',
+                    '#groupCollection > div:nth-child(1) > div > div:nth-child(1)',
```

Comment:
part of the reason i wanted to make these selectors constants is to reduce the brittleness of the E2E tests - could we do that now?

Reply:
Done